site/en/configure/build-performance-breakdown.md

```diff
@@ -0,0 +1,233 @@
+Project: /_project.yaml
+Book: /_book.yaml
+
+# Breaking down build performance
+
+{% include "_buttons.html" %}
+
+Bazel is complex and does a lot of different things over the course of a build,
+some of which can have an impact on build performance. This page attempts to map
+some of these Bazel concepts to their implications on build performance. While
+not extensive, we have included some examples of how to detect build performance
+issues through [extracting metrics](/docs/configure/build-performance-metrics)
```

Comment:
You need to remove `/docs` from these links

Reply:
Done. I'd seen links elsewhere using the `/docs` prefix - are those bugs or do both work with not including the prefix being the preferred way?

Reply:
uff, I might have been mistaken - I will double check every link and leave individual comments